.jazzy.yaml

```diff
@@ -3,9 +3,19 @@ author: Amazon Web Services
 github_url: https://github.com/aws-amplify/amplify-ios
 
 exclude:
-- Pods
+  - Pods
 
 clean: true
 skip_undocumented: false
+undocumented_text: ""
+hide_documentation_coverage: true
 sdk: iphone
 theme: apple
+build_tool_arguments:
+  - -workspace
+  - Amplify.xcworkspace
+  - -scheme
+  - Amplify
```

Comment:
Do these changes update the command we should be using to generate the docs? 

```
// Before
jazzy --output='./docs' --min-acl=public -x -workspace,./Amplify.xcworkspace,-scheme,Amplify
// With this change? Is it possible to put the `output` and `min-acl` over in this yaml file as well?
jazzy --output='./docs' --min-acl=public 
```

Reply:
Yes, these changes make all command line arguments unnecessary.  --output='./docs' and --min-acl=public are the defaults, so they were never actually required unless they were overriding something from the config.

You can now generate the docs simply by running `jazzy` with no args.  However, in order to make the readme pictures show up in the docs, we need to symlink the directory containing them in the docs directory, so something like the following: `jazzy && ln -s ../readme-images docs`.

We should really update the Fastlane release process to generate the docs so that it's not a manual step at all, but in the meantime, I will update the release process wiki once we get these changes merged.